src/studentView.pug

```diff
@@ -0,0 +1,53 @@
+#studentView(hidden='')
+    header
+        #date
+        #account-icon
+            img(src='./gui/img/User.png' alt='User Icon')
+            span#account-username-display Welcome, User
+            #logout
+                span Logout
+    .node-search(ondragover='allowDroppingNode(event)' ondrop='deleteNode(event)')
+        #search-settings
+            input(type='search')
+            span.radioGroup
+                input#search-current(type='radio' name='search-where' value='Search in current module')
+                label(for='search-current') Search Current
+            span.radioGroup
+                input#search-all(type='radio' name='search-where' value='Search in all modules')
+                label(for='search-all') Search All
+        #search-results
+            button.accordion Arithmetic
+            .panel
+                #addition.node.arithmetic.addition(draggable='true' ondragstart='pickUpNode(event)')
```

Comment:
These node objects are repeated a bunch. They should be templated to a single object